app/controllers/v0/sessions_controller.rb

```diff
@@ -108,10 +109,12 @@ def user_login(saml_response)
         after_login_actions
         redirect_to url_service(user_session_form.saml_uuid).login_redirect_url
```

Comment:
Can we just move `callback_stats(:success, saml_response)` to here, out of the conditional, and remove both the new else block and the callback_stats call in login_stats_success?

Reply:
I tried addressing all the calls to `callback_stats` in this commit, but may have gotten a little  carried away.  Let me know what you think, I can revert or apply the same changes to v0.  https://github.com/department-of-veterans-affairs/vets-api/pull/4364/commits/5bf2e03189e9c20303c575b9fc3ced7ff018edf8